spec/requests/document_show_spec.rb

```diff
@@ -2,9 +2,9 @@
 
 require "rails_helper"
 
-RSpec.describe "API request to list planning applications", type: :request, show_exceptions: true do
+RSpec.describe "API request to show document file", type: :request, show_exceptions: true do
   let!(:planning_application) { create(:planning_application, :not_started) }
-  let!(:document) { create(:document, :with_file, planning_application: planning_application) }
+  let!(:document) { create(:document, :with_file, :numbered, planning_application: planning_application) }
```

Comment:
This is a reminder for me to create a chore to change this in the document factory: we don't need this `with_file` trait any more as the basic document factory now contains a file, I should go through and remove these instances. Not for this PR, though.